content/guides/tina-cloud/add-tinacms-to-existing-site/next-changes.md

```diff
@@ -345,6 +345,34 @@ export default function Post({ data, slug}) {
 
 Visit [http://localhost:3000/posts/hello-world](http://localhost:3000/posts/hello-world) to see the GraphQL-powered page.
 
+### Adding contextual editing:
+
+At this point, your page is rendering with the content loaded from Tina. We now want to setup Tina's contextual editing so that you can edit the content via the sidebar.
+
+Let's update your page component like so:
+
+```diff
+- export default function Post({ data, slug}) {
++ export default function Post({ data: initialData, slug, query}) {
```

Comment:
Aren't we suggesting that they no longer need to pass `query` (or `variables) through here? Could also then remove those from the `getStaticProps` return in the code snippet above

Reply:
Yeah technically. I kind of just found I was running into a ton of friction on this page, that I kind of just patched this quick to get it working. I'm going to make a TODO to revisit this guide soon.